src/tool_getparam.c

```diff
@@ -1488,6 +1513,8 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
     case 'D':
       /* dump-header to given file name */
       GetStr(&config->headerfile, nextarg);
+      if(CheckFile(config->headerfile, config->override_file_prohibit))
+          return PARAM_FILE_EXISTS;
```

Comment:
unexpected indent level, we use two spaces

Reply:
^^